src/utils/fetchClient.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const BASE_URL = 'https://mate.academy/students-api';
+const BASE_URL = 'https://mate.academy/students-ap';
```

Comment:
The BASE_URL seems to be incorrect. Please update it to the correct API URL.

Reply:
That was for test, corrected)